src/Audio/audioSceneComponent.ts

```diff
@@ -84,6 +84,12 @@ declare module "../scene" {
          * @see http://doc.babylonjs.com/how_to/playing_sounds_and_music
          */
         headphone: boolean;
+
+        /**
+         * Gets or sets custom audio listener position provider
+         * @see http://doc.babylonjs.com/how_to/playing_sounds_and_music

```

Comment:
Wonderful. Would you be ok to update that doc page to mention your new API?

Reply:
Will do, thanks!

Reply:
@deltakosh done - PR created. Is there going to be a built with this feature in (alpha I assume) that I could fetch and give it a spin in my game?

Reply:
It is on preview already :)

Reply:
@deltakosh is there a way to get preview from npm?

Reply:
nope but I plan to ship it to npm tomorrow ;)

Reply:
@delkatosh oh that's awesome :) it's going to be alpha15 I assume?

Reply:
Correct!